c++/regenerate-bootstraps.sh

```diff
@@ -7,4 +7,5 @@ export PATH=$PWD/bin:$PWD:$PATH
 capnp compile -Isrc --no-standard-import --src-prefix=src -oc++:src \
     src/capnp/c++.capnp src/capnp/schema.capnp \
     src/capnp/compiler/lexer.capnp src/capnp/compiler/grammar.capnp \
-    src/capnp/rpc.capnp src/capnp/rpc-twoparty.capnp src/capnp/persistent.capnp
+    src/capnp/rpc.capnp src/capnp/rpc-twoparty.capnp src/capnp/persistent.capnp \
```

Comment:
Hmm, I wonder if json.capnp really needs to be bootstrapped. Since it's in a separate library, it should be possible to set up the build system to run the newly-built capnp tool to generate this within the build, like we do for all the test files. We couldn't do this with most of the other files because they are actually dependencies of the compiler. Then again, arguably rpc.capnp is not a dependency of the compiler, yet we bootstrap it. I can't quite remember why...

If we do need to go with the bootstrap approach, can you please also update:
* `c++/src/capnp/bootstrap-test.ekam-rule`
* `c++/src/capnp/capnpc.ekam-rule`